tests/store_helpers.py

```diff
@@ -401,7 +401,14 @@ def add_sample(
 
         for key, value in kwargs.items():
             if key == "flowcell":
-                sample.flowcells.append(kwargs["flowcell"])
+                flow_cell: Flowcell = kwargs["flowcell"]
+                metric: SampleLaneSequencingMetrics = store.add_sample_lane_sequencing_metrics(
+                    sample_internal_id=sample.internal_id,
+                    flow_cell_name=flow_cell.name,
+                    **kwargs,
+                )
+                store.session.add(metric)

```

Comment:
This is messy but I couldn't find a clean way that was within the scope of this PR